frontend/components/commons/forms/questions-form/fields/rating/RatingMonoSelection.component.vue

```diff
@@ -65,18 +65,19 @@ export default {
   }
 }
 .label-text {
-  display: flex;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
   width: 100%;
-  border-radius: 50em;
-  height: 32px;
-  background: palette(purple, 800);
+  border-radius: $border-radius-rounded;
+  height: $base-space * 4;
+  min-width: $base-space * 4;
+  padding-inline: $base-space;
   outline: none;
-  padding-inline: 12px;
-  line-height: 32px;
+  background: palette(purple, 800);
+  color: palette(purple, 200);
```

Comment:
from more important to less important => border-radius is less importantthan height/padding, so it's better to put the corresponding css after